src/Files.Backend/Helpers/FileExtensionHelpers.cs

```diff
@@ -26,7 +26,7 @@ public static bool HasExtension(string? filePathToCheck, params string[] extensi
 		/// <returns><c>true</c> if the fileExtensionToCheck is an image;
 		/// otherwise, <c>false</c>.</returns>
 		public static bool IsImageFile(string? fileExtensionToCheck)
-			=> HasExtension(fileExtensionToCheck, ".png", ".bmp", ".jpg", ".jpeg");
+			=> HasExtension(fileExtensionToCheck, ".png", ".bmp", ".jpg", ".jpeg", ".gif", ".tiff");
```

Comment:
You should also include `.tif`, since it is the same as `.tiff`

Reply:
You're absolutely correct, my bad!